src/components/layouts/Footer.jsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
+
 const Footer = () => {
   return (
-    <footer className="flex flex-row items-center text-center justify-center bg-lightGreen h-16 text-black shrink-0">
-      <p>CTD Practicum Team 4</p>
+    <footer className="flex items-center text-center justify-between bg-lightGreen h-16 text-black shrink-0 px-4 lg:px-16 xl:px-14 2xl:px-20">
+      <p>&copy; TalentStudio</p>
+      <p>Designed and developed by Team 4 for Code the Dream</p>
+      <Link to="/team">Meet the Team</Link>
```

Comment:
Please add here the hover effect for the link: hover:text-white hover:underline

Reply:
Thank you, Valentina!
I added the hover effect for the link